uber/site_sections/preregistration.py

```diff
@@ -339,13 +339,12 @@ def process_group_payment(self, session, payment_id, stripeToken):
         else:
             group.amount_paid += charge.dollar_amount
 
-            for attendee in charge.attendees:
+            for attendee in group.attendees:
```

Comment:
The ``uber.utils.Charge`` class takes a list of objects which can be either ``Attendee`` or ``Group`` objects.  If you say ``charge.attendees`` then it will return a list of ``Attendee`` objects which were explicitly passed to the ``Charge`` class.

Whereas if you have a ``group`` which you got out of the ``charge`` and then say ``group.attendees`` then you'll get all attendees which are part of that group, even if they were not explicitly part of the ``Charge`` object.  Since we get these from the database via the foreign keys into the group class, we don't need to say ``session.merge``.

The ``Charge`` class could definitely use some documentation :(